awx/main/scheduler/task_manager.py

```diff
@@ -68,7 +68,7 @@ def after_lock_init(self):
         '''
         Init AFTER we know this instance of the task manager will run because the lock is acquired.
         '''
-        instances = Instance.objects.filter(~Q(hostname=None), capacity__gt=0, enabled=True)
+        instances = Instance.objects.filter(~Q(hostname=None), enabled=True)
```

Comment:
I'm a little worried that this might muck with logic we have for lost/dead instances.